templates/design_system/authorize.html

```diff
@@ -78,7 +78,7 @@ <h2>{{ application.name }} will be able to:</h2>
                 <h2 class="ds-u-color--error">Understand the risks:</h2>
 
                 <p>
-                  You have the right to share your health information, but there may be risks. Be sure to review the app's Privacy Policy and Terms and Conditions. You can revoke an app's access to your data at any time by logging in to your <a href=https://www.mymedicare.gov>MyMedicare.gov</a> account or calling us at 1-800-633-4227.
+                  You have the right to share your health information, but there may be risks. Be sure to review the app's Privacy Policy and Terms and Conditions. You can revoke an app's access to your data at any time by logging in to your <a href=https://www.mymedicare.gov>Medicare.gov</a> account or calling us at 1-800-633-4227.
```

Comment:
```suggestion
                  You have the right to share your health information, but there may be risks. Be sure to review the app's Privacy Policy and Terms and Conditions. You can revoke an app's access to your data at any time by logging in to your <a href=https://www.medicare.gov>Medicare.gov</a> account or calling us at 1-800-633-4227.
```
Just need to update the href.

Reply:
Per the AC, the link stays the same.